packages/react-slider/Spec.md

```diff
@@ -1,63 +1,232 @@
-# @fluentui/react-slider Spec
+# @fluentui/react-slider Draft Component Spec
+
+## Epic Issue
+
+https://github.com/microsoft/fluentui/issues/18886
+
+## Slider Issues
+
+https://github.com/microsoft/fluentui/issues?q=is%3Aissue+is%3Aopen++is%3A+Component%3A+Slider+
 
 ## Background
 
-_Description and use cases of this component_
+The Slider component allows users to quickly select a value (or range) by dragging an icon across a bar. It is often used when setting values with a relaxed precision such as audio volume and screen brightness.
 
 ## Prior Art
 
-_Include background research done for this component_
+### Open UI Research
+
+https://open-ui.org/components/slider.research.parts
+https://open-ui.org/components/slider.research
 
-- _Link to Open UI research_
-- _Link to comparison of v7 and v0_
-- _Link to GitHub epic issue for the converged component_
+#### Research Summary
+
+**Marks**: Amongst other component libraries marks/ticks/notches are used to help visibly differ the current location of the thumb. Marks are also used to create custom steps through providing an array of values to jump too.
 
 ## Sample Code
 
-_Provide some representative example code that uses the proposed API for the component_
+```jsx=
+// Slider can be uncontrolled
+<Slider defaultValue={3} />
+
+// Slider can be controlled
+<Slider value={ 3 } />
+
+// The value can be an array for range slider scenarios
+<Slider value={ [3, 2] } />
+<Slider defaultValue={ [3, 2] } />
```

Comment:
If we are proposing a separate `RangedSlider` we should indicate so in the spec and move this out unto the `RangedSlider` spec.